content/2019-munich/schedule.md

```diff
@@ -61,7 +61,7 @@ title: Schedule
     </td>
   </tr>
   <tr class="break">
-    <td>11:00</td>
+    <td>10:45</td>
```

Comment:
This break is now longer since the rest after it didn't move, but I guess that's the intention so we can give people more time to snack before lunch?

Reply:
Though hmm, I wonder if it'd make sense to make the 12:15 break longer instead (closer to lunch, people will already be more hungry)?

Reply:
There's a longer break after the 3 sets of talks, to allow people to unwind a bit.